lab_2/main.py

```diff
@@ -73,8 +82,10 @@ def fill_lcs_matrix(first_sentence_tokens: tuple, second_sentence_tokens: tuple)
             for j, j_elem in enumerate(second_sentence_tokens):
 
                 if i_elem == j_elem:
-                    f[i][j] = f[i - 1][j - 1] + 1
-
+                    if (j - 1) >= 0 or (i - 1) >= 0:
+                        f[i][j] = f[i - 1][j - 1] + 1
+                    else:
+                        f[i][j] = 1
                 else:
                     f[i][j] = max(f[i][j - 1], f[i - 1][j])
```

Comment:
I agree with lines 86-88, but the same in this` else`, what happens if i == 0 or j == 0?

Reply:
what happens if i_elem != j_elem but i == 0 or j == 0? (same situation here)